cli/tsc.rs

```diff
@@ -288,18 +292,24 @@ impl CompilerConfig {
       _ => b"".to_vec(),
     };
 
+    let content = if let Some(content) = config {
+      let str_ = String::from_utf8(content)?;
+      Some(str_)
+    } else {
+      None
+    };
+
     // If `checkJs` is set to true in `compilerOptions` then we're gonna be compiling
     // JavaScript files as well
-    let compile_js = if let Some(config_content) = config.clone() {
-      let config_str = std::str::from_utf8(&config_content)?;
+    let compile_js = if let Some(config_str) = content.as_ref() {
       CHECK_JS_RE.is_match(config_str)
```

Comment:
we should really be finding a way to refactor this out... Some like this based on the JSON value:

```rust
let mut compile_js = false;
if let Some(value) = compiler_options.as_object().unwrap().get("checkJs") {
  compile_js = value.as_bool().unwrap_or(false);
}
```

Reply:
No worries, will be done, yesterday I was mainly fighting to keep CI green 👍

Reply:
Done